scripts/create_addon

```diff
@@ -56,7 +56,7 @@ pack_addon() {
     fi
     cd $ADDON_BUILD
     echo "*** compressing Addon $PKG_ADDON_ID ... ***"
-    zip -rq $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
+    $TOOLCHAIN/bin/7za a -l -mx9 -bsp0 -bso0 $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
```

Comment:
@vpeter4 `Any special reason for ; character at the end of line?`
it works with or without - the old version had it too, idk if it was some workaround or something (I can drop if you like)